examples/deepspeed_mii/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Running Stable diffusion model using Microsoft DeepSpeed-MII in Torchserve.
```

Comment:
Could you please include a link to the model paper . Also it will be great if you can add few sentences on the motivation